src/test/atlas_planned_maintenance_testing/mod.rs

```diff
@@ -120,7 +121,20 @@ fn write_json(test_runner: &mut TestRunner, mut errors: Vec<Bson>) {
     // The events key is expected to be present regardless of whether storeEventsAsEntities was
     // defined.
     write!(&mut writer, ",\"events\":[").unwrap();
-    test_runner.write_events_list_to_file("events", &mut writer);
+    let event_list_entity = match entities.get("events") {
```

Comment:
I just moved the logic from the function that was previously called here inline.